src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunction.Orchestration.cs

```diff
@@ -13,34 +13,38 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
 using Microsoft.Health.Dicom.Core.Features.Indexing;
+using Microsoft.Health.Dicom.Core.Features.Store;
 using Microsoft.Health.Dicom.Functions.Indexing.Models;
 
 namespace Microsoft.Health.Dicom.Functions.Indexing
 {
     public partial class ReindexDurableFunction
     {
         /// <summary>
-        ///  The orchestration function to add extended query tags.
+        ///  The orchestration function to reindex tags.
         /// </summary>
         /// <param name="context">The context.</param>
         /// <param name="logger">The logger.</param>
         /// <returns>The task.</returns>
-        [FunctionName(nameof(AddAndReindexTagsAsync))]
-        public async Task AddAndReindexTagsAsync(
+        [FunctionName(nameof(ReindexTagsAsync))]
+        public async Task ReindexTagsAsync(
```

Comment:
I've done some additional brainstorming, and I think we can reduce this to 1 orchestration. But for now we can keep it separate as we have more discussions offline lol